client/app/components/composites/MenuMobile/OffScreenMenu.js

```diff
@@ -5,21 +5,26 @@ import css from './MenuMobile.css';
 import MenuSection from './MenuSection';
 import Avatar from '../../elements/Avatar/Avatar';
 import AddNewListingButton from '../../elements/AddNewListingButton/AddNewListingButton';
+import LoginLinks from '../../composites/LoginLinks/LoginLinks';
 
 class OffScreenMenu extends Component {
 
   render() {
     const isOpenClass = this.props.isOpen ? css.offScreenMenuOpen : '';
 
+    const header = this.props.avatar ? [
+      this.props.avatar ? r(Avatar, this.props.avatar) : null,
```

Comment:
As you mention in the latter comment, yes, the earlier code checks for avatarDropdownProps, but even if there are those props defined, this component gets a null in the avatar prop when the user is logged out.